src/Sylius/Behat/Context/Setup/UserContext.php

```diff
@@ -130,6 +130,7 @@ private function createUser(
         $user->setUsername($email);
         $user->setEmail($email);
         $user->setPlainPassword($password);
+        $user->enable();
```

Comment:
IMO it could be useful to make this optional (sometimes you may want to test not enabled users that try to do something).